lib/experimental/fonts/font-library/font-library.php

```diff
@@ -22,19 +22,74 @@
+		),
 		'public'                         => false,
 		'_builtin'                       => true,  /* internal use only. don't use this when registering your own post type. */
-		'label'                          => 'Font Family',
 		'show_in_rest'                   => true,
+		'capabilities'                   => array(
+			'read'                   => 'edit_theme_options',
+			'read_post'              => 'edit_theme_options',
+			'read_private_posts'     => 'edit_theme_options',
+			'create_posts'           => 'edit_theme_options',
+			'publish_posts'          => 'edit_theme_options',
+			'edit_post'              => 'edit_theme_options',
+			'edit_posts'             => 'edit_theme_options',
+			'edit_others_posts'      => 'edit_theme_options',
+			'edit_published_posts'   => 'edit_theme_options',
+			'delete_post'            => 'edit_theme_options',
+			'delete_posts'           => 'edit_theme_options',
+			'delete_others_posts'    => 'edit_theme_options',
+			'delete_published_posts' => 'edit_theme_options',
+		),
+		'map_meta_cap'                   => false,
 		'rest_base'                      => 'font-families',
 		'rest_controller_class'          => 'WP_REST_Font_Families_Controller',
 		'autosave_rest_controller_class' => 'WP_REST_Autosave_Font_Families_Controller',
+		'query_var'                      => false,
 	);
 	register_post_type( 'wp_font_family', $args );
 
+	register_post_type(
+		'wp_font_face',
+		array(
+			'labels'       => array(
+				'name'          => __( 'Font Faces', 'gutenberg' ),
+				'singular_name' => __( 'Font Face', 'gutenberg' ),
+			),
+			'public'       => false,
+			'_builtin'     => true,                              /* internal use only. don't use this when registering your own post type. */
+			'hierarchical' => false,
+			'show_in_rest' => false,
+			'rest_base'    => 'font-faces',
+			'capabilities' => array(
+				'read'                   => 'edit_theme_options',
+				'read_post'              => 'edit_theme_options',
+				'read_private_posts'     => 'edit_theme_options',
+				'create_posts'           => 'edit_theme_options',
+				'publish_posts'          => 'edit_theme_options',
+				'edit_post'              => 'edit_theme_options',
+				'edit_posts'             => 'edit_theme_options',
+				'edit_others_posts'      => 'edit_theme_options',
+				'edit_published_posts'   => 'edit_theme_options',
+				'delete_post'            => 'edit_theme_options',
+				'delete_posts'           => 'edit_theme_options',
+				'delete_others_posts'    => 'edit_theme_options',
+				'delete_published_posts' => 'edit_theme_options',
+			),
+			'map_meta_cap' => false,
+			'query_var'    => false,
+		)
+	);
+
 	// @core-merge: This code will go into Core's `create_initial_rest_routes()`.
 	$font_collections_controller = new WP_REST_Font_Collections_Controller();
 	$font_collections_controller->register_routes();
+
+	$font_faces_controller = new WP_REST_Font_Faces_Controller();
+	$font_faces_controller->register_routes();
```

Comment:
@spacedmonkey We could use your guidance here!

By setting `'show_in_rest' => true`, and using the built in controller instantiation, we could not find a way to disable the autosaves routes, other than overriding with an empty controller (see [font families example](https://github.com/WordPress/gutenberg/blob/4e982f45cc372cae344f05d3d838d6b03fb5596b/lib/experimental/fonts/font-library/class-wp-rest-autosave-font-families-controller.php)).

Is there a better way to do this? Because these post types don't store user content in the way most other post types do, but instead are preconfigured typography settings, I don't think autosaves make sense here.

Reply:
Removed in https://github.com/WordPress/gutenberg/pull/57688/commits/32689e176c2d2df55803a1fa06903ac148e8df7d. Please see the question above about the Autosaves controller.

Reply:
I now see that [attachments are excepted](https://github.com/WordPress/wordpress-develop/blob/e707e37f23dcb6723ba3cea22d45f8f35ef95deb/src/wp-includes/class-wp-post-type.php#L925) from an autosave controller. Perhaps that's how we can handle `wp_font_face` and `wp_font_family`.

Otherwise it looks like specifying an empty class like `stdClass` will also disable autosaves.